cookbook/form/create_form_type_extension.rst

```diff
@@ -129,6 +129,9 @@ The ``alias`` key of the tag is the type of field that this extension should
 be applied to. In your case, as you want to extend the ``file`` field type,
 you will use ``file`` as an alias.
 
+If you want to create a generic feature, applied to all field types, simply
```

Comment:
I followed the vocabulary used in the first section list which says 

> You want to add a **generic feature to several types** (such as adding a "help" text to every field type);

(original emphasis)

But if you still think extension would be a better word, I will be happy to amend my commit.